tests/cypress/e2e/issues_prs2/issue_8952_interpolation_impossible.js

```diff
@@ -0,0 +1,179 @@
+// Copyright (C) 2025 CVAT.ai Corporation
+//
+// SPDX-License-Identifier: MIT
+
+/// <reference types="cypress" />
+
+const taskName = '5frames';
+const labelName = 'label';
+const issueId = 'xxxx';
+const imagesCount = 5;
+const width = 400;
+const height = 400;
+const posX = 50;
+const posY = 50;
+const color = 'white';
+const imageFileName = `image_${issueId}`;
+const archiveName = `${imageFileName}.zip`;
+const archivePath = `cypress/fixtures/${archiveName}`;
+const imagesFolder = `cypress/fixtures/${imageFileName}`;
+const dirToArchive = imagesFolder;
+
+const [fX, fY] = [30, 30];
+const [w, h] = [34, 23];
+const createRectangleShape2Points = {
```

Comment:
2Points / 4Points make sense when you create using UI elements. 
In this case everything created headlessly.